### ISSUE TITLE
Implements changes in EU VAT from 1.1.2019

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/vstore#128]
In most countries (definitely in mine) there is the change in EU VAT system with digital goods.

Now you can decide if you want to use MOSS (in meaning VAT according to customer country) or your business country VAT on all goods (until some hight limit).

So this should be a new preference. 
Apply VAT according to customer country in the EU or not. If not, then always in EU is business VAT applied for not persons. Or companies without VAT.

**Comment by SimSync** [comment]
@Jimmi08 Yes, that's a good idea to add a new pref for this. I'll check it and implement it next time. Thanks!
